doc/classes/Array.xml

```diff
@@ -741,6 +741,14 @@
 				[b]Note:[/b] You should not randomize the return value of [param func], as the heapsort algorithm expects a consistent result. Randomizing the return value will result in unexpected behavior.
 			</description>
 		</method>
+		<method name="swap">
+			<return type="void" />
+			<param index="0" name="first" type="int" />
+			<param index="1" name="second" type="int" />
+			<description>
+				Swaps the elements at indices [param first] and [param second]. If either parameter is negative, the corresponding index is counted from the end of the array. If either index is out of bounds, the array remains unchanged and an error is printed.
```

Comment:
Nitpicks. Plus, "relative to the end of the array" is consistent with the rest.
```suggestion
				Swaps the elements at indices [param first] and [param second]. If either parameter is negative, the corresponding index is relative to the end of the array. If either parameter is out of bounds, the array remains unchanged and an error is generated.
```
or briefer perhaps?
```suggestion
				Swaps the elements at indices [param first] and [param second]. If either parameter is negative, the corresponding index is relative to the end of the array. If either parameter is out of bounds, this method fails.
```

Reply:
I think I'll go with the shorter version. I wrote "index" out of bounds because the parameters are adjusted once to account for negative values, but I don't mind going back to "parameter" instead.